sql/core/src/main/scala/org/apache/spark/sql/execution/aggregate/TungstenAggregationIterator.scala

```diff
@@ -186,77 +182,71 @@ class TungstenAggregationIterator(
       // Partial-only
       case (Some(Partial), None) =>
         val updateExpressions = allAggregateFunctions.flatMap(_.updateExpressions)
-        val algebraicUpdateProjection =
+        val updateProjection =
           newMutableProjection(updateExpressions, aggregationBufferAttributes ++ inputAttributes)()
 
         (currentBuffer: UnsafeRow, row: InternalRow) => {
-          algebraicUpdateProjection.target(currentBuffer)
-          algebraicUpdateProjection(joinedRow(currentBuffer, row))
+          updateProjection.target(currentBuffer)
```

Comment:
For _now_, TungstenAggregationIterator only supports expression-based aggregates, and thus we don't need to explicitly qualify that this is an expression-based update projection. When I refactor TungstenAggregationIterator to support interpreted aggregates as well, I'll re-name to disambiguate.